test/integration/tests/path_block_device_test.rb

```diff
@@ -64,11 +64,11 @@
     end
 
     it 'has no product_version' do
-      file.product_version.must_equal(nil)
+      file.product_version.must_be_nil
     end
 
-    it 'has no file_version' do
-      file.file_version.must_equal(nil)
+    it 'has no product_version' do
+      file.file_version.must_be_nil
```

Comment:
the description talks about product_version vs the test looking at file version 😉